setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Create a virtual environment and install python dependencies
+if [ ! -d "venv" ]; then
+    python3 -m venv venv
```

Comment:
I recall setting having to set the python version specifically to 3.6 in some of our other repos (properly-util-python, pipeline-access, feature-property-api).  Would that be required here too?

Reply:
I will cover this in a following PR.